src/lang/en.json

```diff
@@ -562,6 +562,49 @@
       "custodianOfRecords": "Custodian of Records"
     }
   },
+  // TODO: MOVE this to the shared layer when upgrading to that layer
+  "filingTypes": {
+    "adminFreeze": "Freeze Business",
+    "adminUnfreeze": "Unfreeze Business",
+    "agmLocationChange": "AGM Location Change",
+    "alteration": "Alteration",
+    "annualReport": "Annual Report {AGM_YEAR}",
+    "amalgamationApplication": "Amalgamation Application",
+    "amalgamationApplicationSubTypes": {
+      "horizontal": "Amalgamation Application Short-form (Horizontal)",
+      "regular": "Amalgamation Application (Regular)",
+      "vertical": "Amalgamation Application Short-form (Vertical)"
+    },
+    "changeOfAddress": "Address Change",
+    "changeOfCompanyInfo": "Company Information Change",
+    "changeOfDirectors": "Director Change",
+    "changeOfName": "Legal Name Change",
+    "changeOfOfficers": "Officer Change",
+    "changeOfRegistration": "Change of Registration",
+    "continuationIn": "Continuation Application",
+    "continuationAuthorization": "Continuation Authorization",
+    "conversion": "Record Conversion",
+    "correction": "Correction",
+    "courtOrder": "Court Order",
+    "dissolution": "Dissolution",
+    // TODO: dissolutionSubtypes
```

Comment:
Would it make sense to handle `dissolutionTypeToName` here, now?

If not, please change "TODO" (which means to do in this PR) to "FUTURE" (which means later). Thx.

Reply:
I added the types there (note that it was a todo before) I did not however handle the function as I wasn't sure of the difference between "Involuntary Dissolution" and "Failed to File"